src/lib/temperature-bounds.ts

```diff
@@ -23,34 +23,26 @@ export const getHeatBounds = ({
   mode,
   minHeat,
   maxHeat,
-  minCool,
```

Comment:
Why is `minCool` special?

Reply:
Can you elaborate?

Reply:
Resolved over huddle, @xplato could use your eyes here for sure